src/Azure.Functions.Cli/Kubernetes/KubectlHelper.cs

```diff
@@ -135,16 +135,25 @@ async Task TimeoutFunc()
                 }
             }
 
+            static string MaybeNamespace(string @namespace)
+            {
+                if (string.IsNullOrWhiteSpace(@namespace) == false)
+                {
+                    return $" --namespace {@namespace}";
+                }
+                return " --namespace default";
```

Comment:
I think this should return `string.Empty` to use the default namespace from kubeconfig rather than `default`, right?

Reply:
Jup, should be ommited then indeed

Reply:
done.